src/strategies/property.sml

```diff
@@ -6,45 +6,91 @@ import "strategies.property_importance";
 
 signature PROPERTY =
 sig
+    exception ParseError;
+
+    type pKind;
+    datatype pValue = Label of string | Number of int | Boolean of bool;
```

Comment:
Same as above, consider just making it `datatype value` (or `datatype propertyValue`).